upgrade: Backend repo mapping for repochecks

skazi0 opened the pull request:
Backend provides mapping product to repositories in the repochecks
responses. It can be used instead of the PRODUCTS_REPO_CHECKS_MAP
constant.

skazi0 committed:
upgrade: Backend repo mapping for admin repochecks

Backend provides mapping product to repositories in the repochecks
responses. It can be used instead of the PRODUCTS_REPO_CHECKS_MAP
constant.

skazi0 commented:
![](https://github.trello.services/images/mini-trello-icon.png) [crowbarctl should show nice names for missing repositories](https://trello.com/c/xr2UOZyN/261-crowbarctl-should-show-nice-names-for-missing-repositories)

Itxaka reviewed (approved):
Tested and works

MaximilianMeister reviewed (approved):
as far as I can judge

skazi0 committed:
upgrade: Backend repo mapping for nodes repochecks

Backend provides mapping product to repositories in the repochecks
responses. It can be used instead of the PRODUCTS_REPO_CHECKS_MAP
constant.

MaximilianMeister reviewed (changes requested):
You need to remove the `Cloud` repo again, see https://github.com/crowbar/crowbar-core/pull/1006#issuecomment-274067570

skazi0 committed:
upgrade: Removed Cloud repo from the repos list

skazi0 commented:
@MaximilianMeister `Cloud` repo removed
